src/model_config_utils.h

```diff
@@ -279,4 +279,20 @@ TRITONSERVER_DataType DataTypeToTriton(const inference::DataType dtype);
 /// \return The data type.
 inference::DataType TritonToDataType(const TRITONSERVER_DataType dtype);
 
+/// Check if the model configs are only differ in instance group.
+/// \param old_config The old model config.
+/// \param new_config The new model config.
+/// \return The comparison result.
```

Comment:
Add an explanation of what a return value of `true` vs. `false` is in the comment. 

```suggestion
/// \return TRUE if model configurations are the same not checking instance_group; FALSE if one or more fields, excluding instance_group, are not the same.
```

Reply:
Update commit: [Improve model_config_utils comments](https://github.com/triton-inference-server/core/pull/189/commits/97f7c89c3e06e26d15ce7e05589723f6ac97c0c5)